src/Workspaces/Core/Portable/Classification/ClassificationContext.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Classification
+{
+    /// <summary>
+    /// A context for aggregating classifications from one or more <see cref="ClassificationProvider"/>'s.
+    /// </summary>
+    internal class ClassificationContext
+    {
+        private readonly List<ClassifiedSpan> _spans;
```

Comment:
The list is coming from a pool so it gets reused.

---
In reply to: [78663266](https://github.com/dotnet/roslyn/pull/13741#discussion_r78663266) [](ancestors = 78663266)